okapi-core/src/test/java/org/folio/okapi/bean/BeanTest.java

```diff
@@ -233,7 +233,7 @@ public void testModuleDescriptorBadMethod() {
       + "    \"id\" : \"sample\"," + LS
       + "    \"version\" : \"1.0\"," + LS
       + "    \"handlers\" : [ {" + LS
-      + "      \"methods\" : [ \"GET\", \"NOST\" ]," + LS
+      + "      \"methods1\" : [ \"GET\", \"POST\" ]," + LS
```

Comment:
is having an invalid property value checked elsewhere?

Reply:
Adding one for RoutingEntryTest .. However HttpMethod accepts any value, so Okapi doesn not reject, say, method=GYF

Reply:
I reverted this and fixed setMethod to throw exception for bad method.